egs/gigaspeech/ASR/local/compute_fbank_gigaspeech.py

```diff
@@ -86,6 +80,19 @@ def get_parser():
         "pre-computation might currently consume excessive memory and time "
         "-- use on-the-fly feature extraction in the training script instead.",
     )
+    parser.add_argument(
+        "--num-workers",
+        type=int,
+        default=4,
+        help="Number of dataloading workers used for reading the audio.",
+    )
+    parser.add_argument(
+        "--batch-duration",
```

Comment:
Also, there is another parameter:
https://github.com/lhotse-speech/lhotse/blob/master/lhotse/features/kaldifeat.py#L158
```python3
    chunk_size: Optional[int] = 1000
```

You can use
```
extractor = KaldifeatFbank(
            KaldifeatFbankConfig(device="cuda", chunk_size=<some_value>),
        )
```
to limit the chunk size. I am not sure whether your OOM error is related to these two parameters.

Reply:
1000 is a reasonable default, you shouldn't get OOM with that value.

Reply:
[posting on web as email unreliable] There are also commands in lhotse to split and recombine data.  I forget the names/invocation but Piotr answered my question on this in either this repo or lhotse's repo.

Reply:
It is in https://github.com/lhotse-speech/lhotse/issues/452#issuecomment-962402670